.taskcluster.yml

```diff
@@ -79,6 +79,9 @@ tasks:
         'public/bin/mozilla/logins-release-{{ event.version }}.aar':
           type: 'file'
           path: '/build/application-services/logins-api/android/library/build/outputs/aar/logins-release.aar'
+        'public/bin/mozilla/places-release-{{ event.head.tag }}.aar':
```

Comment:
I think you really want `event.version` here.  (The underlying GH events are different.)